projects.json

```diff
@@ -878,7 +878,23 @@
       {
         "version": "4.2",
         "commit": "759c1088a1d9ba94f44009295fd17c20b1375524"
-      }
+      },
+      {
+        "version": "5.0",
```

Comment:
@groue Ok I did some digging, I see that https://github.com/apple/swift-source-compat-suite/pull/613 removed the revisions for 5.0, 5.1, and 5.3 by mistake.  

The source compat suite will pass the language major version to the compiler. So the swift build will use `-swift-version 5` for each of these GRDB versions you listed, meaning each entry will be built the same way (although with different repository revisions), I'm not sure if that's what you intended. If you want to build all these various GRDB revisions the same way (with `-swift-version 5`), then leave it as is. 

If you want to build just one revision of GRDB with `-swift-version 5`, I'd recommend just one entry with your most recent commit, namely 
```
       {
         "version": "5.7",
         "commit": "2d19cab2b525f885ee8d7efec96093431f6b5f20"
       }
```

It's up to you, as I'm not sure about the nature of the commit revisions in question

Reply:
Thank you @justice-adams-apple :-)

> It's up to you, as I'm not sure about the nature of the commit revisions in question

Thanks for the precision :-) The version I care about is the latest, but the library has changed a lot in years, so I'd rather keep a few old versions here - for the sake of compatibility testing :-)

I have pushed a fix, e1e956ab2b64f1c8d2efe1a3ac6de3f279a3b42d, with the latest commit supported by each language version.